core/backend/ios/scripts/processAppCenterLogs.js

```diff
@@ -0,0 +1,106 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) 2022 Bentley Systems, Incorporated. All rights reserved.
```

Comment:
Can you switch this to the correct header?

Reply:
Fixed.